modules/olm-disabling-copied-csvs.adoc

```diff
@@ -12,6 +12,13 @@ When Operators are configured to use the `AllNamespaces` install mode, versus ta
 
 To support these larger clusters, cluster administrators can disable copied CSVs for Operators installed with the `AllNamespaces` mode.
 
+[WARNING]
+====
+If you disable copied CSVs, users can only discover Operators that are both available and installed directly in their namespace. 
+
+This behavior occurs because copied CSVs are used to identify the Operators available for a given namespace. A user's role-based access control (RBAC) limits the use of the OperatorHub page in the console. As a result, users with insufficient RBAC cannot see Operators configured to reconcile events in their namespace for Operators installed in separate namespaces.
+====
```

Comment:
> users with insufficient RBAC cannot see Operators configured to reconcile events in their namespace for Operators installed in separate namespaces

I am not clear on what this means, but that might be me. If you are comfortable with it, I cannot argue.